RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplate.cc

```diff
@@ -2687,9 +2715,9 @@ int SiPixelTemplate::qbin(int id, float cotalpha, float cotbeta, float qclus)
    // Interpolate for a new set of track angles
    
    // Local variables
-   float pixmx, sigmay, deltay, sigmax, deltax, sy1, dy1, sy2, dy2, sx1, dx1, sx2, dx2, locBz, locBx; //  lorywidth, lorxwidth;
+   float pixmx, sigmay, deltay, sigmax, deltax, sy1, dy1, sy2, dy2, sx1, dx1, sx2, dx2, locBz; //  lorywidth, lorxwidth;
    // Local variables
-   locBx = 1.f;
+   float locBx = 1.f;
```

Comment:
Certainly not an issue, but why to define float locBx here, differently for what you do for locBy?

Reply:
makes no difference at all. no reason.